torchgen/model.py

```diff
@@ -511,6 +530,14 @@ def from_yaml(
                     raise AssertionError(f"illegal tag {t}")
         assert isinstance(tags, set)
 
+        composite_s = e.pop("composite", None)
+        assert composite_s is None or isinstance(composite_s, str)
+        composite = (
```

Comment:
this PR doesn't allow you to do any of the composite logic for structured operators, right? Maybe you should assert that if `composite` is provided, then the operator isn't structured?